docker-compose.yml

```diff
@@ -12,11 +12,11 @@ services:
   webserver:
     image: puckel/docker-airflow:1.10.1
```

Comment:
Should we change this image version to 1.10.9, too?

Reply:
Sounds reasonable, but won't the build details override the image details? (Making this line redundant)
Or am I missing something here?